spec/dummy/db/migrate/20200408161720_create_groups.rb

```diff
@@ -3,6 +3,7 @@ def change
     create_table :groups do |t|
       t.string :display_name, null: false
       t.string :email, null: false
+      t.boolean :random_attribute, default: false
```

Comment:
Why wouldn't these be in new migrations?

Reply:
In a real rails application, yes. For a dummy test application, doesn't really matter